src/__tests__/lib/containers/SynapseTable.test.tsx

```diff
@@ -685,65 +678,57 @@ describe('basic functionality', () => {
       })
       it('renders a user card link', () => {
         const tableCell = shallow(
-          <div>
-            {renderTableCell({
-              ...tableCellProps,
-              mapUserIdToHeader,
-              colIndex: USERID_INDEX,
-              columnValue: mockUserCardValue,
-            })}
-          </div>,
+          <SynapseTableCell
+            {...tableCellProps}
+            colIndex={USERID_INDEX}
+            columnValue={mockUserCardValue}
+            mapUserIdToHeader={mapUserIdToHeader}
+          />,
         )
         expect(tableCell.find(UserCard)).toHaveLength(1)
       })
       it('renders a markdown value', () => {
         const mockMarkdownColumnValue = '# column markdown'
-        const tableCell = shallow(
-          <div>
-            {renderTableCell({
-              ...tableCellProps,
-              colIndex: USERID_INDEX,
-              columnValue: mockMarkdownColumnValue,
-              mapUserIdToHeader,
-              columnLinkConfig: {
-                isMarkdown: true,
-                matchColumnName: 'a',
-              },
-              columnName: 'a',
-              selectColumns: [
-                { columnType: EntityColumnType.STRING, name: 'a', id: '' },
-              ],
-            })}
-          </div>,
+        const tableCell = mount(
+          <SynapseTableCell
+            {...tableCellProps}
+            colIndex={USERID_INDEX}
+            columnValue={mockMarkdownColumnValue}
+            mapUserIdToHeader={mapUserIdToHeader}
+            columnLinkConfig={{
+              isMarkdown: true,
+              matchColumnName: 'a',
+            }}
+            columnName={'a'}
+            selectColumns={[
+              { columnType: EntityColumnType.STRING, name: 'a', id: '' },
+            ]}
+          />,
         )
         expect(tableCell.find(MarkdownSynapse).props().markdown).toEqual(
           mockMarkdownColumnValue,
         )
       })
```

Comment:
Since we were calling a function, rather than rendering a component, we weren't actually doing shallow rendering in any of these tests! This is the only one that required changing to `mount`, since it inspects the MarkdownSynapse component